saleor/graphql/context.py

```diff
@@ -28,6 +28,7 @@ def get_context_value(request: HttpRequest) -> SaleorContext:
 
 def clear_context(context: SaleorContext):
     context.dataloaders.clear()
+    del context.user
```

Comment:
As you're deleting the assigned user here, don't you also need to `del` the potential app?
Also, what in a situation where `context.user` is None?

Reply:
We don't need to delete the `app`. The user is wrapped in `SimpleLazyObject`. The reference cycle is generated via `SimpleLazyObject`. When a user is `None` it's still wrapped with `SimpleLaysObject`.

Reply:
Ach I see. Based on the typing in SaleorContext, I assumed that this is "plain" user